Makefile

```diff
@@ -0,0 +1,88 @@
+# TODO: run golint, errcheck, staticcheck, unused
```

Comment:
and ineffassign?

I assume these aren't in the target because there are too many failures in the codebase. Is that right?

Reply:
Yes, that's right. I left the individual targets so people could more easily chip away at the issues when time permits.